javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,25 @@
+const class07Students = [];
+
+function addStudentToClass(studentName) {
+   
+  if (studentName === 'Queen Margaret'|| studentName === 'Queen margaret' || studentName === 'queen margaret' || studentName === 'queen Margaret'){
```

Comment:
Are you sure this is what requirements of this funtcion say? 
but assuming that it was that: is there any other way of checking that? Hint: string.split() and string.toLowerCase()